aspnet/publishing/iis.rst

```diff
@@ -126,7 +132,7 @@ Troubleshooting:
 
 - If your application uses the `.UseUrls(...)` extension on `WebHostBuilder`, make sure you have positioned the `.UseUrls(...)` extension before the `.UseIISIntegration()` extension on `WebHostBuilder`. `.UseIISIntegration()` must overwrite any values you provide in `.UseUrls(...)` in order for the reverse-proxy to succeed.
 
-Issue #2
+Issue #3
```

Comment:
Yeah, at the time the issues were a single combined bulleted list, which didn't render very well. Also each issue needs it's own linkable subheading so that they can be referenced. Unfortunately I didn't have time to come up with subheading text before the RC2 release needed to go out. @GuardRex If you could help out with subheading text that would be great.